### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by Shifra-fraiman**
all forms and rtl and generic form

### CODE REVIEW

File: src/components/Forms/BusinessForm/BussinesForm.tsx
Comment (ChayaKook) [inline src/components/Forms/BusinessForm/BussinesForm.tsx:1]: What is the purpose of the notice?

File: src/components/Providers/HebrewDirectionProvider.tsx
Comment (ChayaKook) [inline src/components/Providers/HebrewDirectionProvider.tsx:1]: You are talented!

File: src/pages/HomePage/HomePage.tsx
Comment (ChayaKook) [inline src/pages/HomePage/HomePage.tsx:1]: Role: Never not tuch in page that not in your own. Is that acceptable to you?
if not, please write me...

**Review by ChayaKook — COMMENTED** [review]
perfect job! but please read my comments.

File: src/components/Providers/HebrewDirectionProvider.tsx
Comment (Shifra-fraiman) [reply src/components/Providers/HebrewDirectionProvider.tsx:1]: fun!

**Pull request merged**